.github/workflows/add-issues-to-projects.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto Assign to Project
+
+on:
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add PRs to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/altinn/projects/3
+          github-token: ${{ secrets.ASSIGN_PROJECT_TOKEN }}
```

Comment:
Can this not use the default `secrets.GITHUB_TOKEN`? It should have [access to `repository-projects` by default](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) , but maybe thats not enough?

Reply:
Not sure. This was something that @acn-sbuad created and I just copied, and I assume @altinnadmin was involved when deciding on the secret usage. I thought it was a dedicated token with just enough access to do the job.

Reply:
@RonnyB71 @haakemon This is the reason: https://docs.github.com/en/issues/trying-out-the-new-projects-experience/automating-projects#github-actions-workflows
![image](https://user-images.githubusercontent.com/6088624/175290828-4b798a5e-b8bc-4579-85ad-d1863841af1f.png)

This is ONLY a temporary solution until we get a GitHub App up and running, ref. what @tjololo have been testing.
If this is not in a backlog somewhere it needs to be created.

FYI @acn-sbuad